website/docs/advanced/tenancy.md

```diff
@@ -11,37 +11,65 @@ This feature is in alpha. Use at your own risk.
 ::::
 
 ::::info
-This feature is available from 2024.2 and is not to be confused with brands, which used to be called tenants.
+This feature is available from 2024.2 and is not to be confused with [brands](../core/brands.md), which were previously called tenants.
 ::::
 
-## Preparations
+## About tenants
 
-Starting with 2024.2, authentik allows an administrator or operator to create multiple tenants. This means that an operator can manage several authentik installations without having to deploy additional instances.
+Starting with version 2024.2, authentik allows an administrator or operator to create multiple tenants. This means that an operator can manage several different and distinct authentik installations, each with it's own Install ID and license(s). The relationships between tenant and installation (Install ID) is a 1:1 relationship, so for each tenant, there is a unique Install ID.
```

Comment:
The terms "installation" and "instance" don't have any meaning to me.  To me, an "instance" is a running server; something that can be horizontally scaled to meet an authentik licensee's need. In cloud speak, a collection of instances is called a "service" (since it's a bunch of servers performing a single role in an overall deployment).  I'm not sure what an "installation" is in this terminology.

Reply:
Yes, that's exactly why I rewrote that section, to remove the confusion between "installation" and "Instance." Note that the word "instance" no longer appears in the new content.

Reply:
Ah, good!